### DISCUSSION TITLE
Login token

### DISCUSSION THREAD

**Discussion by ivsaneesh** [vendurehq/vendure#1584] [Q&A] [answered]
I am not getting token when using  login or authentication api. Any advice

**Comment by michaelbromley** [comment]
Hi,

Please provide more information: 

- What's you config for `authOptions.tokenMethod`?
- What query are you executing?
- What is the response?
- What are the response headers?

**Comment by ivsaneesh** [comment]
In config token used is 
 . tokenMethod: 'bearer'

    authOptions: {
        superadminCredentials: {
            identifier: 'superadmin',
            password: 'superadmin',
        },
        cookieOptions: {
          secret: process.env.COOKIE_SECRET || 'cookie-secret',
        },
        tokenMethod: 'bearer',
        requireVerification: false,
    },



![Screenshot 2022-05-26 0000](https://user-images.githubusercontent.com/34297370/170415306-ced51390-e153-4b42-b108-069b527653bb.jpg)

**Reply by michaelbromley** [reply]
OK. The guide for how to get and manage the bearer token is here: https://www.vendure.io/docs/storefront/managing-sessions/#bearer-token-sessions

**Reply by ru0ya** [reply]
The page link brings an error 404, is there an updated link for this?

**Reply by michaelbromley** [reply, marked as answer]
https://docs.vendure.io/guides/storefront/connect-api/#managing-sessions
